network-api/networkapi/wagtailcustomization/tests.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+from wagtail.contrib.redirects.models import Redirect
+
+
+class LocalizedRedirectTests(TestCase):
+    def test_plain_redirect(self):
+        """
+        Test that the base redirect works.
+        """
+        Redirect.objects.create(
+            old_path='/test',
+            redirect_link='http://example.com'
+        )
+        response = self.client.get('/test')
```

Comment:
This doesn't work in so many ways I posted https://stackoverflow.com/questions/65082910/how-to-create-a-wagtail-redirect-during-testing to hopefully understand why.